widget/embedded/src/components/Slippage/Slippage.tsx

```diff
@@ -65,6 +68,7 @@ export function Slippage() {
           max="30"
           step="0.01"
           fullWidth
+          style={{ width: '127px' }}
```

Comment:
I think giving it a range would be better. I mean `min-width` and `max-width`. 
I also not a fan of inline-styling, you can create a container and add your constraints to that.